### PR TITLE
e2e: fix vm-pipe-to-file

### PR DESCRIPTION
**Pull request by askervin**
Fixes regression caused by double-quoting `$append` in commit a60565a69b4a2e548198a699cdbe63bd069b6ab2.

This causes failing proxy env setup when setting up new VM.

### CODE REVIEW

File: demo/lib/vm.bash [hunk 1]
```diff
@@ -365,12 +365,12 @@ vm-pipe-to-file() { # script API
     #   echo 'Ahoy, Matey...' | vm-pipe-to-file /etc/motd
     local tmp append
     tmp="$(mktemp vm-pipe-to-file.XXXXXX)"
-    if [ "$1" = "--append" ]; then
+    if [ "$1" == "--append" ]; then
```
Comment (marquiz) [inline demo/lib/vm.bash:368]: Why this change to double equal signs?

Comment (askervin) [reply demo/lib/vm.bash]: That seems to be a bad habit - even man page says that `=` is to be used with `[`. Thanks!

Comment (askervin) [reply demo/lib/vm.bash]: Fixed

**Review by marquiz — APPROVED** [review]
lgtm

File: demo/lib/vm.bash (hunk 1 above)
Comment (marquiz) [reply demo/lib/vm.bash]: I think both work in bash. But single equal sign is the POSIX standard or smth
